crates/astria-utils/src/config.rs

```diff
@@ -0,0 +1,17 @@
+use figment::{
+    providers::Env as FigmentEnv,
+    Figment,
+};
+use serde::{
+    Deserialize,
+    Serialize,
+};
+
+pub trait AstriaConfig<'a>: Serialize + Deserialize<'a> {
```

Comment:
Bikeshed: maybe just call this trait `Config`?

Reply:
Since the individual structs are called `Config` I wanted to avoid calling this `Config` - plus the name `AstriaConfig` invokes an Astria specific config structure, which I like